wasm-engines/docker/fizzy.Dockerfile

```
@@ -0,0 +1,5 @@
+FROM jwasinger/llvm-10:1.0
+
+# install fizzy
+RUN git clone https://github.com/wasmx/fizzy.git --single-branch --branch master
```

Use this for now:
```suggestion
RUN git clone https://github.com/wasmx/fizzy.git --single-branch --branch v0.1.0
```

Should use v0.2.0 in a few days.

---

`wasm-engines/fizzy.sh` breaks with this release of fizzy.  `ewasm/fizzy:1.0` currently uses `e6187e90fdacc2fd4c21432d2a4ec0ea90250417` (May 31st commit) which works with the script.  We could merge this and open an issue (which I would work on after this branch is merged).

---

What script? These scripts seem to be really brittle.

---

https://github.com/ewasm/benchmarking/blob/master/wasm-engines/fizzy.sh